src/components/BrowsingMode/BrowsingMode.tsx

```diff
@@ -43,13 +43,13 @@ export function BrowsingModeMenu({ closeMenu }: { closeMenu?: () => void }) {
 
   return (
     <div id="browsing-mode">
-      <Stack spacing="md" className="sm:min-w-96">
+      <Stack spacing="md" classNames="sm:min-w-96" py={"xl"}>
```

Comment:
Using `classNames` won't work as intended, this is more a Mantine prop
```suggestion
      <Stack spacing="md" className="sm:min-w-96" py="xl">
```